lib/internal/sync/controller/note_controller.dart

```diff
@@ -20,8 +19,7 @@ class NoteController implements NoteInterface {
   }
 
   String toJson(Note note) {
-    SyncNote savingNote = SyncNote.fromNote(note);
-    return json.encode(savingNote);
+    return json.encode(Utils.toSyncMap(note));
```

Comment:
why does this even exist at all

Reply:
I will remove it, it used to be more lines